### PR TITLE
docs(alert): responsiveness improvements

### PR DESCRIPTION
**Pull request by portikM**
# Summary

Before:
Screen Shot 2023-03-06 at 5 58 26 PM

After:
Screen Shot 2023-03-06 at 5 59 23 PM

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: src/components/KAlert/KAlert.vue [hunk 1]
```diff
@@ -116,6 +116,7 @@
 
       <KButton
         v-if="dismissType === 'button'"
+        class="white-space-nowrap"
```
Comment (adamdehaven) [inline src/components/KAlert/KAlert.vue:119]: question:
Shouldn't a KButton, by default, have `white-space: nowrap`?

suggestion: I think this rule should just be added directly to KButton instead?

Also, let's not keep adding utility classes. Again, as we move towards an actual design system, all of these utility classes will have to be refactored out of Kongponents (and utilize different, targeted classes) so let's not add more in the meantime if not absolutely necessary.

Comment (portikM) [reply src/components/KAlert/KAlert.vue]: Yeah you're right, better handle that from KButton itself. Removed the newly added utility class too (keep forgetting we don't want to add new ones at all).

**Review by adamdehaven — APPROVED** [review]
LGTM -- you'll definitely want to manually pull this into `khcp-ui` and test locally to ensure the button CSS change isn't breaking

**Comment by portikM** [comment]
> LGTM -- you'll definitely want to manually pull this into `khcp-ui` and test locally to ensure the button CSS change isn't breaking

Do you think we have any occurrences of KButton where text wraps in Konnect?

**Comment by adamdehaven** [comment]
> Do you think we have any occurrences of KButton where text wraps in Konnect?

There are definitely places where they wrap _unintentionally_ -- so I think it will be fine; I'm just saying click around and spot-check 👍🏼

**Pull request merged**